static/elements/chromedash-roadmap.js

```diff
@@ -159,6 +160,9 @@ class ChromedashRoadmap extends LitElement {
     milestoneNumsArray.forEach((milestoneNum) => {
       newMilestonesInfo[milestoneNum].version = milestoneNum;
       newMilestonesInfo[milestoneNum].features = milestoneFeatures[milestoneNum];
+      Object.keys(newMilestonesInfo[milestoneNum].features).forEach(status => {
+        newMilestonesInfo[milestoneNum].features[status].sort( (a, b) => compareFeatures(a, b));
```

Comment:
I think the last part of this line could just be `sort(compareFeatures)`.
Same thing for three other occurrences below.